apps/user/routes.coffee

```diff
@@ -78,58 +77,48 @@ fetchFocus = (user, per=4)->
       res.locals.sd.BLOCKS = blocks.toJSON()
       res.render "index", author: res.locals.author, blocks: blocks.models
 
-@userChannelsByAlpha = (req, res, next) ->
+@index = (req, res, next) ->
+
   return next() unless res.locals.author
 
   channels = new UserBlocks null,
     user_slug: req.params.username
     subject: 'channels'
 
-  Q.all [
-    channels.fetchUntilEnd data: auth_token: req.user?.get('authentication_token')
-    fetchFocus res.locals.author, req.query.per
-  ]
-  .then ([response, focus]) ->  
-    res.locals.sd.FOCUS = focus?.toJSON()
+  channels.fetchUntilEnd 
```

Comment:
That wasn't my invention (though `fetchUntilEndInParallel` was). Either way—is there an alternative?